modules/react/tabs/stories/examples/DynamicTabs.tsx

```diff
@@ -32,9 +32,9 @@ export const DynamicTabs = () => {
    * @param id The id of the item that will be removed
    */
   const removeItem = <T extends unknown>(id: string, model: ReturnType<typeof useTabsModel>) => {
-    const index = model.state.items.findIndex(item => model.getId(item) === model.state.cursorId);
+    const index = model.state.items.findIndex(item => item.id === model.state.cursorId);
     const nextIndex = index === model.state.items.length - 1 ? index - 1 : index + 1;
-    const nextId = model.getId(model.state.items[nextIndex]);
+    const nextId = model.state.items[nextIndex].id;
```

Comment:
>  Optional function to return an id of an item. If not provided, the default function will return
>      * the `id` property from the object of each item.

So the default behavior is that each item has an id property, so no need to call `getId`